lambdas/file_access_patterns/lambda_for_fileDelete/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+* This lambda takes the userId and fileId and deletes the corresponding file, updating the storage_used for the user with the given userId and delete all the corresponding URLs for the deleted file.
```

Comment:
avoid using long single-line sentences in the docstring, instead break the line at sufficient length to make the code concise and readable.